utils/gtfTobed.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import argparse
+import subprocess
+import glob
+
+def main():
+    parser = argparse.ArgumentParser(description='GTF to BED')
+    parser.add_argument('--gtf', dest='gtf', type=str,
+                        help='Input gtf file.', required=True)
+    args = parser.parse_args()
```

Comment:
good point. Don't we need to know the output name anyway, at least for wiring in Snakemake? I mean, I can see the output name will be the same as input, just bed instead of gtf, but if we provide this util as a docker image, the naming will not be transparent, will it?

Reply:
Agree, I think it's better to be able to define the output file name otherwise users will have to dig into the Dockerfile to find the expected output (especially for Snakemake as you say)

Reply:
I have added an `--out_bed` argument to indicate the name of the output bed file